packages/generator-converters/src/wirePropertyV2ToSdkPropertyDefinition.ts

```diff
@@ -97,7 +97,7 @@ function objectPropertyTypeToSdkPropertyDefinition(
     case "array":
       return objectPropertyTypeToSdkPropertyDefinition(propertyType.subType);
     case "timeseries":
-      if (propertyType.itemType.type === "string") {
+      if (propertyType.itemType?.type === "string") {
```

Comment:
Item type is now optional in the API because sensor types can be either string or number